types/canvasjs/canvasjs-tests.ts

```diff
@@ -1,248 +1,246 @@
+import { CanvasJS } from "canvasjs";
```

Comment:
According to its docs, CanvasJS is a global, not a module. (Every single example just accesses a global `CanvasJS` object)

This line should not be needed, and should be removed.